### DISCUSSION TITLE
Docker problem

### DISCUSSION THREAD

**Discussion by ghost** [monkeytypegame/monkeytype#3739] [Questions] [answered]
I am trying to set up a development environment using Docker. I have no issues with the backend but I can't run the frontend. The terminal output:

> Attaching to monkeytype-frontend
> monkeytype-frontend  | 
> monkeytype-frontend  | > monkeytype@1.14.3 dev-fe
> monkeytype-frontend  | > cd frontend && npm run dev
> monkeytype-frontend  | 
> monkeytype-frontend  | 
> monkeytype-frontend  | > monkeytype-frontend@1.14.3 dev
> monkeytype-frontend  | > concurrently "webpack serve --config=./webpack/config.dev.js" "npx gulp watch"
> monkeytype-frontend  | 
> monkeytype-frontend  | sh: 1: concurrently: not found
> monkeytype-frontend exited with code 127

Any advice how to fix it?

**Comment by Miodec** [comment]
And youve got the latest code correct

**Reply by ghost** [reply]
Yes, I synced and pulled.

**Reply by Miodec** [reply]
Hm, im not sure. Concurrently is included as a dependency in both frontend, root and backend.

**Comment by rizwanmustafa** [comment, marked as answer]
You need to run npm run install-all in the root directory. Then you need to run docker. You haven't got concurrently installed yet.

**Reply by MaximilienGora** [reply]
Correct!

**Reply by ghost** [reply]
Thank you.
